stubs/passpy/passpy/store.pyi

```diff
@@ -0,0 +1,28 @@
+from collections.abc import Iterable, Iterator, Mapping
+from typing import Match
+
+class Store:
+    def __init__(
+        self,
+        gpg_bin: str = ...,
+        git_bin: str = ...,
+        store_dir: str = ...,
+        use_agent: bool = ...,
+        interactive: bool = ...,
+        verbose: bool = ...,
+    ) -> None: ...
+    def __iter__(self) -> Iterator[str]: ...
+    def is_init(self) -> bool: ...
+    def init_store(self, gpg_ids: Iterable[str], path: str | None = ...) -> None: ...
```

Comment:
The code has
```
        # Ensure that gpg_ids is a list so that the later .join does
        # not accidentally join single letters of a string.
        if gpg_ids is not None and not isinstance(gpg_ids, list):
            gpg_ids = [gpg_ids]
```
So I think it can be None, str, or list[str], but not any Iterable.